### ISSUE TITLE
undefined reference to "make_timeout_time_ms"

### ISSUE THREAD

**Issue by lingmaple** [sandeepmistry/pico-lorawan#34]
![185b6e2835888609428d613f5499d2a](https://user-images.githubusercontent.com/47233485/229527474-d6dd002b-72cf-41a6-bec0-50de9bd51e6e.png)
please

**Comment by jerryneedell** [comment]
Is your copy of the library up to date?  This was fixed by PR #33


This can be fixed by adding including "pico/time.h" in  src/lorawan.c

```
diff --git a/src/lorawan.c b/src/lorawan.c
index 9e4cec3..dc24e60 100644
--- a/src/lorawan.c
+++ b/src/lorawan.c
@@ -28,6 +28,7 @@
 #include <string.h>
 
 #include "pico/lorawan.h"
+#include "pico/time.h"
 
 #include "board.h"
 #include "rtc-board.h"
```

**Comment by sandeepmistry** [comment]
@lingmaple did you get a changes to try @jerryneedell's suggestion above (https://github.com/ArmDeveloperEcosystem/lorawan-library-for-pico/issues/34#issuecomment-1494472699) ?
